helm_tests/airflow_aux/test_basic_helm_chart.py

```diff
@@ -27,7 +27,7 @@
 
 from tests.charts.helm_template_generator import render_chart
 
-OBJECT_COUNT_IN_BASIC_DEPLOYMENT = 35
+OBJECT_COUNT_IN_BASIC_DEPLOYMENT = 38
```

Comment:
With rpc being disabled by default, I don't think we need any of the changes in this file.